src/libmongoc/libmongoc.rc.in

```diff
@@ -34,7 +34,7 @@ BEGIN
             VALUE "OriginalFilename", MONGOC_OUTPUT_BASENAME "-" MONGOC_API_VERSION ".dll"
             VALUE "ProductName", "MongoDB C Driver"
             VALUE "ProductVersion", MONGOC_VERSION_S
-            VALUE "LegalCopyright", "Copyright (C) 2013-present MongoDB, Inc."
```

Comment:
Any reason for leaving the start year as 2013?

Reply:
Good catch. I may have accidentally reverted the update to year in a local rebase. Fixed.